x/tss/types/codec.go

```diff
@@ -7,53 +7,21 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
 	reward "github.com/axelarnetwork/axelar-core/x/reward/exported"
-	"github.com/axelarnetwork/axelar-core/x/tss/tofnd"
 )
 
 // RegisterLegacyAminoCodec registers concrete types on codec
 func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&HeartBeatRequest{}, "tss/HeartBeatRequest", nil)
-	cdc.RegisterConcrete(&StartKeygenRequest{}, "tss/StartKeygen", nil)
-	cdc.RegisterConcrete(&ProcessKeygenTrafficResponse{}, "tss/KeygenTraffic", nil)
-	cdc.RegisterConcrete(&ProcessSignTrafficRequest{}, "tss/SignTraffic", nil)
-	cdc.RegisterConcrete(&RotateKeyRequest{}, "tss/RotateKey", nil)
-	cdc.RegisterConcrete(&VoteSigRequest{}, "tss/VoteSig", nil)
-	cdc.RegisterConcrete(&VotePubKeyRequest{}, "tss/VotePubKey", nil)
-	cdc.RegisterConcrete(&RegisterExternalKeysRequest{}, "tss/RegisterExternalKey", nil)
-	cdc.RegisterConcrete(&SubmitMultisigPubKeysRequest{}, "tss/SubmitMultisigPubKeys", nil)
-	cdc.RegisterConcrete(&SubmitMultisigSignaturesRequest{}, "tss/SubmitMultisigSignatures", nil)
 }
 
 // RegisterInterfaces registers types and interfaces with the given registry
 func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
 	registry.RegisterImplementations((*sdk.Msg)(nil),
 		&HeartBeatRequest{},
-		&StartKeygenRequest{},
```

Comment:
These should be registered under `app/codec.go` for `axelar.tss.v1beta1.*` to not break tx parsing

Reply:
will add